sycl/doc/UsersManual.md

```diff
@@ -290,6 +290,28 @@ and not recommended to use in production environment.
 
     NOTE: This flag is currently only supported with the CUDA and HIP targets.
 
+
+**`-f[no]-sycl-esimd-force-stateless-mem`** [EXPERIMENTAL]
+
+    Enforces usage of stateless memory accesses and automatic conversion of
+    "stateful" memory accesses via SYCL accessors to "stateless" accesses
+    within ESIMD (Explicit SIMD) kernels by compiler.
+
+    This option disables the intrinsics and methods accepting SYCL accessors
```

Comment:
It disables in -f or -fno- form? Can you clarify here?

Reply:
-f option disables those intrinsics that cannot be converted automatically. I updated the text. Thank you.